data/news/san_mateo.json

```diff
@@ -3,6 +3,13 @@
   "title": "San Mateo County COVID-19 News",
   "home_page_url": "https://cmo.smcgov.org/press-releases",
   "items": [
+    {
+      "id": "12711 at https://cmo.smcgov.org",
+      "url": "https://cmo.smcgov.org/press-release/sept-25-2020-health-officer-order-prohibits-removal-fire-debris-burn-sites-pending",
+      "title": "Sept. 25, 2020: Health Officer Order Prohibits Removal of Fire Debris from Burn Sites Pending Development of State and Federal Process",
+      "date_published": "2020-09-25T22:05:08Z",
+      "summary": "Sept. 25, 2020San Mateo County Health Officer Dr. Scott Morrow has issued a health order prohibiting the unsafe removal, transport, and disposal of fire debris and other hazardous materials from structures burned during the CZU August Lightning Complex Fires without written permission from Environmental Health Services."
```

Comment:
We should do something about the date getting prefixed to the summary. It looks like it has no space separating it from the text in the source data, too, which is not great.

Reply:
Tacked this onto sfbrigade/data-covid19-sfbayarea#134, since it fits in the same set of issues.